doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -14,9 +14,15 @@ GNU ARM Embedded
 
    .. warning::
 
-      Like the Zephyr repository, do not install the toolchain in a directory
-      with spaces anywhere in the path. On Windows, we'll assume you install
-      into the directory :file:`C:\\gnu_arm_embedded`.
+      Do not install the toolchain into a path with spaces. On
+      Windows, we'll assume you install into the directory
+      :file:`C:\\gnu_arm_embedded`.
+
+   .. warning::
+
+      The latest release of the Windows toolchain has a critical bug,
```

Comment:
Can we add a little bit of text what that critical bug is.

Reply:
I believe that a user that wants to install a toolchain is not interested in the details of bugs in other revisions of the toolchain.

The critical bug is referenced in the commit message, documented in a Zephyr issue, and exists as an upstream gcc bug report. This is enough IMHO.

Reply:
I guess I could add an rst comment.

Reply:
Added a comment.